### PR TITLE
Remove unused api-proxy from all endpoints

### PR DESCRIPTION
**Pull request by vfarah-if**
## Description of change

Remove unused api-proxy from all endpoints

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [x] Has the branch been rebased to master?
- [x] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [x] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

File: src/apps/companies/apps/exports/client/ExportWins/tasks.js [hunk 1]
```diff
@@ -69,8 +69,8 @@ export function fetchExportWins({ companyId, companyName, activePage }) {
   const offset = activePage * 10 - 10
   const param = offset ? '?offset=' + offset : ''
 
-  return axios
-    .get(`/api-proxy/v4/company/${companyId}/export-win${param}`)
+  return apiProxyAxios
+    .get(`/v4/company/${companyId}/export-win${param}`)
     .catch((e) => {
       if (e.response?.status === 501) {
```
Comment (vfarah-if) [inline src/apps/companies/apps/exports/client/ExportWins/tasks.js:75]: Is this still applicable?

File: src/apps/companies/apps/exports/client/ExportWins/tasks.js [hunk 2]
```diff
@@ -69,8 +69,8 @@ export function fetchExportWins({ companyId, companyName, activePage }) {
...
         return { [NOT_IMPLEMENTED]: true }
```
Comment (vfarah-if) [inline src/apps/companies/apps/exports/client/ExportWins/tasks.js:76]: Is this still applicable

File: src/client/components/RoutedAdvisersTypeahead/index.jsx [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 
 import { parseAdviserData } from '../../../common/formatAdviser'
+import { apiProxyAxios } from '../Task/utils'
 
 const fetchAdvisers = (onlyShowActiveAdvisers) => {
   return throttle((searchString) => {
     if (searchString.length) {
-      return axios
+      return apiProxyAxios
         .get('/api-proxy/adviser/', {
```
Comment (cgsunkel) [inline src/client/components/RoutedAdvisersTypeahead/index.jsx:16]:
```suggestion
        .get('/adviser/', {
```

File: src/client/modules/Interactions/CollectionList/tasks.js [hunk 4]
```diff
@@ -62,7 +61,7 @@ const getInteractions = ({
   company_one_list_group_tier,
   dit_participants__team,
 }) =>
-  axios
+  apiProxyAxios
     .post('/api-proxy/v3/search/interaction', {
```
Comment (cgsunkel) [inline src/client/modules/Interactions/CollectionList/tasks.js:65]:
```suggestion
    .post('/v3/search/interaction', {
```

**Review by cgsunkel — APPROVED** [review]
I spotted a couple of remaining `/api-proxy/` usages but apart from that this LGTM

**Review by santoshdasa12345 — APPROVED** [review]

**Pull request merged**